superset/charts/api.py

```diff
@@ -509,20 +516,14 @@ def thumbnail(
             FileWrapper(screenshot), mimetype="image/png", direct_passthrough=True
         )
 
-    def add_apispec_components(self, api_spec: APISpec) -> None:
-        for chart_type in CHART_DATA_SCHEMAS:
-            api_spec.components.schema(
-                chart_type.__name__, schema=chart_type,
-            )
-        super().add_apispec_components(api_spec)
-
```

Comment:
Are these now populated automatically?

Reply:
moved it to base, we can now set these kind of things using `openapi_spec_component_schemas` class attr.